examples/interactive.jl

```diff
@@ -739,7 +739,10 @@ end
         RecordEvents(scene, @replace_with_a_path)
     end
 
-    @cell "Ambient Occlusion" [SSAO, shading, interaction] begin
+    @cell "Ambient Occlusion" [SSAO, ssao, shading, interaction] begin
+        using GLMakie
+        GLMakie.enable_SSAO[] = true
```

Comment:
This should best be an AbstractPlotting option, since WGLMakie will also support SSAO!